Data/Ascii/test/AsciiReaderTest.cpp

```diff
@@ -62,14 +85,14 @@ BOOST_AUTO_TEST_CASE(TestMomentumEnergyOrder) {
 }
 
 BOOST_AUTO_TEST_CASE(TestOverwrite) {
-  const char *Filename = "Data_AsciiReaderTest-TestOverwrite.dat";
+  const char *FileName = "Data_AsciiReaderTest-TestOverwrite.dat";
```

Comment:
Why use a const char* here (also the std::remove below)?

Reply:
Yeah `string` is better. The remove is just to clean the output after running `ctest`. Should I delete it?

Reply:
Yeah it gets deleted after running out of scope anyways

Reply:
No, `remove` deletes the output file

Reply:
[`std::remove`](https://en.cppreference.com/w/cpp/io/c/remove) requires `const char*` for some reason...

Reply:
I see, nvm then